pkg/monitor/sqsevent/sqs-monitor_test.go

```diff
@@ -144,6 +170,43 @@ func TestMonitor_EventBridgeSuccess(t *testing.T) {
 	}
 }
 
+func TestMonitor_EventBridgeTestNotification(t *testing.T) {
+	msg, err := getSQSMessageFromEvent(asgLifecycleTestNotification)
+	h.Ok(t, err)
+	messages := []*sqs.Message{
+		&msg,
+	}
+	sqsMock := h.MockedSQS{
+		ReceiveMessageResp: sqs.ReceiveMessageOutput{Messages: messages},
+		ReceiveMessageErr:  nil,
+	}
+	dnsNodeName := "ip-10-0-0-157.us-east-2.compute.internal"
+	ec2Mock := h.MockedEC2{
+		DescribeInstancesResp: getDescribeInstancesResp(dnsNodeName, true, true),
+	}
+	drainChan := make(chan monitor.InterruptionEvent, 1)
+
+	sqsMonitor := sqsevent.SQSMonitor{
+		SQS:              sqsMock,
+		EC2:              ec2Mock,
+		ManagedAsgTag:    "aws-node-termination-handler/managed",
+		ASG:              mockIsManagedTrue(nil), // TODO: is this the right mock factory function?
```

Comment:
Has this TODO been resolved?

Reply:
Yes, I just forgot to remove the comment.